src/components/parts/_legacy/main-article/komponenter/ArtikkelDato.tsx

```diff
@@ -68,7 +71,7 @@ const ArtikkelDato = (props: Props) => {
             {publishedString}
             {modifiedString && (
                 <>
-                    <span aria-hidden="true">{' |'}</span>
+                    <span aria-hidden="true">{' | '}</span>
```

Comment:
Er det virkelig behov for forskjellig behandling av newsPress og ikke her ...?

Reply:
nei :)